www/css/cytube.css

```diff
@@ -107,13 +96,6 @@
     cursor: pointer;
 }
 
-#userlist, #messagebuffer {
```

Comment:
userlist and message buffer were moved when moving css and are now on line 25-44. My apologies